dlp/snippets/inspect/inspect_test.go

```diff
@@ -545,3 +545,20 @@ func TestInspectImageFileListedInfoTypes(t *testing.T) {
 		t.Errorf("inspectImageFileListedInfoTypes got %q, want %q", got, want)
 	}
 }
+
+func TestInspectBigQueryTableWithSampling(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	topicID := "go-lang-dlp-test-bigquery-with-sampling-topic"
+	subscriptionID := "go-lang-dlp-test-bigquery-with-sampling-subscription"
+
+	var buf bytes.Buffer
+	if err := inspectBigQueryTableWithSampling(&buf, tc.ProjectID, topicID, subscriptionID); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+	if want := "Job Created"; !strings.Contains(got, want) {
```

Comment:
This only seems to test that the first half of the sample executes successfully. Is that okay?

Reply:
Updated the test case as well.